src/pages/blog/airship-update-august-2020.md

```diff
@@ -0,0 +1,98 @@
+---
+templateKey: blog-post
+title: Airship Update - August 2020
+author: Alexander Hughes
+date: 2020-08-13T12:00:00.000Z
+category:
+  - id: category-C98iZYrE1
+    label: Airship 2.0
+---
+
+What follows is the Airship Community Update for August 2020, brought to you by the Airship Technical Committee.
+<!-- more -->
+
+## **SPOTLIGHT ON...**
+
+- On 22-July at the [_OpenDev: Hardware Automation_](https://www.openstack.org/events/opendev-2020/opendev-schedule-2)
```

Comment:
Will everyone know what _the_ baremetal whitepaper is? A small preview of the whitepaper and its relevance to airship could go a long way. Something like:

_Several members of the Airship community contributed to a whitepaper recently released by the OpenStack Foundation that details the OpenDev community's investment in open source bare metal provisioning._

Reply:
Done